app/assets/stylesheets/dashboard.css.scss

```diff
@@ -38,10 +36,74 @@
       padding:     20px 0 30px 0;
     }
 
+    h2.headline {
+      font-family: Neuton;
+      color:       #192023;
+      font-size:   45px;/* Approximation due to font substitution */
+      font-weight: 300;
+      line-height: 70px;/* Approximation due to font substitution */
+      text-align:  center;
+      padding:     50px 100px 30px 100px;
+
+      a {
+        text-decoration: underline;
+        color: #54d1b8;
+      }
+    }
+
+    .banner {
+      background:       #54d1b8;
+      color:            #edf12c;
+      font-family:      Roboto;
+      font-size:        22px;
+      font-weight:      400;
+      padding:          30px 100px;
+
+      a {
+        color:          #edf12c;
+        text-decoration: underline;
+        font-weight:      700;
+      }
+
+      iron-icon {
```

Comment:
I figured it might not but then the icon showed up `¯\_(ツ)_/¯` (see the screenshot above). I'll try and find another icon set we can use in the Rails templates.